mod/file/download.php

```diff
@@ -4,7 +4,12 @@
  * 
  * @package ElggFile
  */
-require_once __DIR__ . '/../../vendor/autoload.php';
```

Comment:
Oh nevermind, I misread what it was doing - I thought you were skipping the require of the file if it didn't exist